projects/packages/forms/src/contact-form/css/grunion.css

```diff
@@ -80,6 +83,38 @@
 	margin: 0;
 }
 
+/** New Phone Field Styles */
```

Comment:
instead of add this here. Lets add it to a `style.scss` that is closer to the element. That get included both on the frontend as well as in the editor. See `input-rating/style.scss`. 
This would make it also possible to ship less css when the page loads if the phone field is not used at all.

Reply:
Mmm... I'll get back to you on this, some things are worth keeping together when they address a wide range of elements. You're likely right, just let me check.